src/common/liboptparse/optparse.h

```diff
@@ -130,6 +130,12 @@ struct optparse_subcommand {
  */
 #define OPTPARSE_OPT_HIDDEN       0x2
 
+/*
+ *  Apply optional argument to short options as well as long. The default
+ *   is that optional arguments only apply to longopts.
+ */
+#define OPTPARSE_OPT_SHORTOPT_OPTIONAL_ARG 0x4
```

Comment:
Nice flag name :-)

Making fun of names is easy, thinking of a better one...more difficult.  I've got nothing.

Reply:
Yeah, the optional short option opt is too long :slightly_frowning_face: